src/main/java/org/apache/mesos/scheduler/DefaultScheduler.java

```diff
@@ -81,63 +79,67 @@ void awaitTermination() throws InterruptedException {
     }
 
     Plan getPlan() {
```

Comment:
Good that you brought this one up. I was debating between:
1. `getDeployPlan`
2. `getPlans` (which would query PlanManagers, collect and return Plans)